crates/spicepod/src/component.rs

```diff
@@ -16,7 +17,7 @@ pub struct ComponentReference {
     pub r#ref: String,
 
     #[serde(skip_serializing_if = "Vec::is_empty")]
-    #[serde(rename = "dependsOn", default)]
+    #[serde(rename = "dependsOn", alias = "datasets", default)]
```

Comment:
reuse component reference for model too.

Reply:
`dependsOn` is a different concept - we shouldn't use it to represent what datasets the model depends on

Reply:
Actually maybe this is fine.